rs/ic_os/attestation/testing/BUILD.bazel

```diff
@@ -0,0 +1,20 @@
+load("@rules_rust//rust:defs.bzl", "rust_library")
+
+package(default_visibility = ["//visibility:public"])
+
+DEPENDENCIES = [
```

Comment:
Could you inline these? We're moving away from this pattern of having `DEPENDENCIES` in a variable because it makes thing harder to read and encourages accidentally introducing unused dependencies when the variable is shared